legal-api/tests/unit/services/test_authorization.py

```diff
@@ -575,7 +576,7 @@ def mock_auth(one, two):  # pylint: disable=unused-argument; mocks of library me
          ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'], 'general', [BASIC_USER], False),
 
         ('user_active', Business.State.ACTIVE, 'consentContinuationOut', None,
-         ['BC', 'BEN', 'ULC', 'CC'], 'staff', [BASIC_USER], False),
+         ['BC', 'BEN', 'ULC', 'CC'], 'general', [BASIC_USER], True),
```

Comment:
It was my understanding that CCOs can only be generated by staff. Has that changed (or am I mistaken)?

Reply:
It's a new requirement